go/enclave/rollupchain/rollup_chain.go

```diff
@@ -150,66 +150,62 @@ func (rc *RollupChain) isGenesisBlock(block *types.Block) bool {
 
 //  STATE
 
-// Recursively calculates the block state, all the logs in the chain so far, and new logs for the given block.
-func (rc *RollupChain) updateState(b *types.Block) (*obscurocore.BlockState, []*types.Log, []*types.Log) {
```

Comment:
I removed the return of all the logs across the entire historical chain. We only want the new logs now (for use when creating a block submission response).